fabfile/run.py

```diff
@@ -203,3 +202,33 @@ def tpch_queries(query_info, connectionURI, pg_version, citus_version, config_fi
         out_val = run(run_string)
         results_file.write(out_val)
         results_file.write('\n')
+
+def filter_and_put_citus_valgrind_outputs(valgrind_log_path):
+    run('cat {} | grep citus.so | awk \'{ print $1 }\' | uniq  > trace_ids'.format(valgrind_log_path))
```

Comment:
might be useful to add a comment that explains what this line does

Reply:
done